cli/js/permissions.ts

```diff
@@ -68,6 +68,19 @@ export class Permissions {
     const { state } = sendSync(dispatch.OP_REVOKE_PERMISSION, desc);
     return new PermissionStatus(state);
   }
+
+  /** Requests the permission.
+   *       const status = await Deno.permissions.request({ name: "env" });
+   *       if (status.state === "granted") {
+   *         home = Deno.homeDir();
+   *       } else {
+   *         console.log("'env' permission is denied.");
+   *       }
+   */
+  async request(desc: PermissionDescriptor): Promise<PermissionStatus> {
+    const { state } = sendSync(dispatch.OP_REQUEST_PERMISSION, desc);
+    return new PermissionStatus(state);
+  }
```

Comment:
This needs to be added to lib.deno_runtime.d.ts

Reply:
Slight error in the example too. I got this when trying to run it:
```
 ️⚠️  Deno requests to access to environment variables.. Grant? [g/d (g = grant, d = deny)] g
error: Uncaught ReferenceError: home is not defined
► file:///Users/rld/src/deno/x.js:3:8

3   home = Deno.homeDir();
         ^

    at file:///Users/rld/src/deno/x.js:3:8
```

Reply:
Thank you for catching!